src/frontend/templates/components/Filters.js

```diff
@@ -71,12 +71,12 @@ export const Filters = () => {
                 <div className="px-xs pt-xs">
                     <label className='d-flex px-xs ai-c gap-1'> 
                     <input onChange={() =>  filtersDispatcher({ type: 'sort', payload: { ...filtersState.sort , sortBy: 'highlow' } })}  
-                    id='highlow' type='radio' name='sort' /> 
+                    id='highlow' type='radio' name='sort' checked={filtersState.sort.sortBy === 'highlow'} /> 
                     Sort High To Low 
                     </label> 
                     <label className='d-flex px-xs ai-c gap-1'> 
                     <input onChange={() =>  filtersDispatcher({ type: 'sort', payload: { ...filtersState.sort , sortBy: 'lowhigh' } })}  
-                     id='lowhigh' type='radio' name='sort' /> 
+                     id='lowhigh' type='radio' name='sort' checked={filtersState.sort.sortBy === 'lowhigh'} /> 
                     Sort Low to High
                     </label>       
```

Comment:
Suggestion: You can save the `sortBy` values as constants and then import them in this Filters file.
It would help your code editor to provide suggestions for this variable, increasing productivity

Reply:
> Suggestion: You can save the `sortBy` values as constants and then import them in this Filters file. It would help your code editor to provide suggestions for this variable, increasing productivity
thanks for the suggestion Om will surely work in further iterations to implement that